src/test/scala/com/databricks/spark/avro/AvroWriteBenchmark.scala

```diff
@@ -19,14 +19,17 @@ package com.databricks.spark.avro
 import java.sql.Date
 import java.util.concurrent.TimeUnit
 
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
+import scala.math.BigDecimal.RoundingMode
 import scala.util.Random
+
 import com.google.common.io.Files
 import org.apache.commons.io.FileUtils
+
 import org.apache.spark.sql._
 import org.apache.spark.sql.types._
 
-import scala.math.BigDecimal.RoundingMode
+// scalastyle:off println
```

Comment:
?

Reply:
```Scala
// scalastyle:off println
...
// scalastyle:on println
```

Reply:
This is for turning off `println` check for the rest of this scala file.